clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/ArraysTests.scala

```diff
@@ -1192,4 +1193,69 @@ trait ArraysTests { self: BaseIntegrationSpec =>
       storedDocument1.unsafeGet[String]("notes") should be("I'm a note")
     }
   }
+
+  it should "amend documents with new metadata" in {
```

Comment:
Note that this doesn't test the use case, which is, "update all arrays records with regulatory designation X". We'd probably want to only run that as an integration test, since we wouldn't want to run it in dev.